ods_ci/tests/Tests/1200__ai_explainability/1203__trustyai_drift_metrics.robot

```diff
@@ -0,0 +1,89 @@
+*** Settings ***
+Documentation     This is a resource file for running Drift Metrics.
+Suite Setup       Prepare TrustyAi-tests Test Suite
+Suite Teardown    Teardown TrustyAi-tests Test Suite
+Library           OperatingSystem
+Library           Process
+Resource          ../../Resources/Common.robot
+Resource          ../../Resources/RHOSi.resource
+
+
+*** Variables ***
+${TRUSTYAI-TESTS_URL}                 %{TRUSTYAI-TESTS-URL=https://github.com/trustyai-explainability/trustyai-tests}
+${TRUSTYAI-TESTS_DIR}                 trustyai-tests
+${GIT_BRANCH}                         main
+${PYTHON_VERSION}                     3.12
+${VIRTUAL_ENV_NAME}                   venv${PYTHON_VERSION}
+${TESTS_FOLDER}                       drift
+${LOG_OUTPUT_FILE}                    test-output/pytest-logs/drift_test.txt
+
+
+*** Test Cases ***
+Run Drift Metrics Tests
+    [Documentation]    Verifies that the Bias metrics are available in Metrics Console for a model which was
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    RHOAIENG-8163     Smoke
+    Run Drift Pytest Framework
+    ${result} =   Run Process  source ${VIRTUAL_ENV_NAME}/bin/activate && cd ${TRUSTYAI-TESTS_DIR} && poetry env use 3.9 &&  poetry install && poetry run pytest -v -s trustyai_tests/tests/${TESTS_FOLDER}/
+    ...    shell=true
+    ...    stderr=STDOUT
```

Comment:
isn't this already run in `Run Drift Pytest Framework`?

Reply:
Removed now